modules/akamaiDapRtdProvider.js

```diff
@@ -53,60 +61,95 @@ export function addRealTimeData(rtd) {
  * Real-time data retrieval from Audigent
  * @param {Object} reqBidsConfigObj
  * @param {function} onDone
- * @param {Object} rtdConfi
+ * @param {Object} rtdConfig
  * @param {Object} userConsent
  */
 export function getRealTimeData(bidConfig, onDone, rtdConfig, userConsent) {
+  let entropyDict = JSON.parse(localStorage.getItem(DAP_CLIENT_ENTROPY));
+  let loadScriptPromise = new Promise((resolve, reject) => {
+    if (rtdConfig && rtdConfig.params && rtdConfig.params.dapFpTimeout && Number.isInteger(rtdConfig.params.dapFpTimeout)) {
+      setTimeout(reject, rtdConfig.params.dapFpTimeout, Error('DapFP script could not be loaded'));
+    }
+    if (entropyDict && entropyDict.expires_at > Math.round(Date.now() / 1000.0)) {
+      logMessage('Using cached entropy');
+      resolve();
+    } else {
+      if (typeof window.dapCalculateEntropy === 'function') {
+        window.dapCalculateEntropy(resolve, reject);
+      } else {
+        if (rtdConfig && rtdConfig.params && rtdConfig.params.dapFpUrl) {
+          let fpScript = document.createElement('script');
+          fpScript.setAttribute('src', rtdConfig.params.dapFpUrl);
+          fpScript.onload = () => dapUtils.dapGetEntropy(resolve, reject);
+          window.document.body.appendChild(fpScript);
```

Comment:
You have to use our methods to load a script

Reply:
Hi @patmmccann ,
I tried to follow what ftrackIdSystem.js module is doing to import the script.
Are you suggesting that I should use adLoader.loadExternalScript to load the script ? If we want to do that, then we need to modify adloader as well to add our module in _approvedLoadExternalJSList.
If you are suggesting something else then could you please point me to it ?

Reply:
you're correct I opened https://github.com/prebid/Prebid.js/issues/8421 to address

Reply:
https://github.com/prebid/Prebid.js/commit/2a05213fb73d88ac939c0b0dd4b5750649446372 seems to be an example